docs/guides/Package-authoring-best-practices.md

```diff
@@ -0,0 +1,177 @@
+---
+title: Package authoring best practices
+description: A general guide of best practices for creating high quality NuGet packages.
+author: chgill-MSFT
+ms.author: chgill
+ms.date: 09/17/20
+ms.topic: conceptual
+---
+
+# Package authoring best practices
```

Comment:
Should we list some of the most important things that NuGet already has pack warnings about?  For example, things related to PackageReference compatibility. I'm not 100% sure we have a pack warning for all of these but:
* tools\install.ps1 and tols\uninstall.ps1 are not compatible with PackageReference
* If using `content\`, also include `contentFiles\`
* If adding msbuild props\targets to your package, understand `build\` vs `buildTransitive\`, and prefer `build\<tfm>\packageid.props` rather than `build\packageid.props`.

Reply:
I think we should also consider anything that is a best practice & doesn't warn today as a way we can also enhance the packing experience.